addons/gut/gui/gut_config_gui.gd

```diff
@@ -7,7 +7,9 @@ class DirectoryCtrl:
 		get:
 			return get_text()
 		set(val):
-			set_text(val)
+			text = val
```

Comment:
Remove the assignment this assignment.  It will make it clear that this is just a wrapper around `_txt_path.text`.